src/inputs/Table/ColumnTypes.js

```diff
@@ -30,58 +33,76 @@ const _boolSetter = (params) => {
 const _dateSetter = (params) => {
   const dateFormat = params.context.dateFormat;
   let newValue = params.newValue;
-  if (!params.context.editMode || !DateUtils.isValid(newValue, dateFormat)) {
-    newValue = params.oldValue;
+  const allowedEmptyFields =
+    params.context.editMode && params.colDef.cellEditorParams?.acceptsEmptyFields && newValue.length === 0;
+  if (allowedEmptyFields) {
+    params.data[params.colDef.field] = newValue;
+    return true;
   } else {
-    const minValue = params.column.userProvidedColDef?.cellEditorParams?.minValue;
-    const maxValue = params.column.userProvidedColDef?.cellEditorParams?.maxValue;
-    if (minValue !== undefined) {
-      newValue = DateUtils.strMax(newValue, minValue, dateFormat) || params.oldValue;
-    }
-    if (maxValue !== undefined) {
-      newValue = DateUtils.strMin(newValue, maxValue, dateFormat) || params.oldValue;
+    if (!params.context.editMode || !DateUtils.isValid(newValue, dateFormat)) {
+      newValue = params.oldValue;
+      params.data[params.colDef.field] = newValue;
+      return true;
```

Comment:
I'm not sure we should have a `return true` in this case, because it prevents the call to 
`params.data[params.colDef.field] = DateUtils.format(DateUtils.parse(newValue, dateFormat), dateFormat);` that was performed before this branch.